bokehjs/src/coffee/common/embed.coffee

```diff
@@ -8,6 +8,18 @@ HasProperties = require "./has_properties"
 {pull_session} = require "./client"
 {Promise} = require "es6-promise"
 
+_handle_notebook_comms = (msg) ->
+  logger.debug("handling notebook comms")
+  doc = _.values(@)[0] # should only be one
```

Comment:
hmm. it may be that the comms target should be in the render item (like sessionid) so there can be a different one for each document in principle. Also it's analogous to sessionid here perhaps, we could handle it in a very similar way.